src/app/listener.ts

```diff
@@ -29,6 +29,18 @@ export class Listener {
     });
 
     this.container.clientService.on('ready', async () => {
+      // Enforce only one server
+      if (this.container.clientService.guilds.cache.size > 1) {
+        this.container.loggerService.error(
+          'More than one server detected. This bot is intented to run in only one server, ' +
+            'and can cause issues when in multiple.\n Please remove this bot from other ' +
+            'servers, or create a new application for Lion.'
+        );
+
+        this.container.clientService.destroy();
```

Comment:
why not just throw Error? the program shouldnt be running if we're in this state

Reply:
Destroy kills the client, I cant throw an unhandled error and crash it instead if you want